### PR TITLE
[sailfish-browser] Add descriptions for select test case. JB#51225

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

File: tests/manual/testselect.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <title>TestSelect</title>
   </head>
   <body>
+    <h1>Single select without group headers<h1>
```
Comment (llewelld) [inline tests/manual/testselect.html:14]: Should be `</h1>` to close in all cases, I think.

**Review by llewelld — CHANGES_REQUESTED** [review]
The headings are clear, but just the end tags need correcting.

File: tests/manual/testselect.html (hunk 1 above)
Comment (rainemak) [reply tests/manual/testselect.html:14]: Sure... too quickie :+1:

**Review by llewelld — APPROVED** [review]
Perfect, thanks :+1:

**Pull request merged**